examples/cfd/01_convection.ipynb

```diff
@@ -261,23 +264,25 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "dt*(-1.0*u(t, x, y)/h_y + 1.0*u(t, x, y - h_y)/h_y - 1.0*u(t, x, y)/h_x + 1.0*u(t, x - h_x, y)/h_x + u(t, x, y)/dt)\n"
+      "(-dt*h_x*u(t, x, y) + dt*h_x*u(t, x, y - h_y) - dt*h_y*u(t, x, y) + dt*h_y*u(t, x - h_x, y) + h_x*h_y*u(t, x, y))/(h_x*h_y)\n"
      ]
     }
    ],
    "source": [
     "from devito import solve\n",
+    "from sympy import nsimplify\n",
     "\n",
     "stencil = solve(eq, u.forward)\n",
     "\n",
-    "print(stencil)"
+    "# nsimplify: 1.0*x = x\n",
+    "print(nsimplify(stencil))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The careful reader will note that this is now the equivalent symbolic expression to the RHS term of the NumPy code we showed earlier - only with `dx` and `dy` denoted as `h` and `dt` denoted as `s`, while `u(t, x, y)`, `u(t, x - h, y)` and `u(t, x, y - h)` denote the equivalent of $u^{n}_{i, j}$, $u^{n}_{i-1, j}$ and $u^{n}_{i, j-1}$ respectively. \n",
+    "The careful reader will note that this is now the equivalent symbolic expression to the RHS term of the NumPy code we showed earlier - only with `dx` and `dy` denoted as `h_x` and `h_y` respectively, while `u(t, x, y)`, `u(t, x - h, y)` and `u(t, x, y - h)` denote the equivalent of $u^{n}_{i, j}$, $u^{n}_{i-1, j}$ and $u^{n}_{i, j-1}$ analytically and `u[1:, 1:]`, `u[:-1, 1:]` and `u[1:, :-1]` in the numpy implementation respectively. \n",
```

Comment:
'...this is now the equivalent symbolic expression to the RHS term of the...' -> '...this is equivalent to the symbolic expression for the RHS of the..."